spec/models/comment_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+describe Comment do 
+
+  include TestFactories
+
+  describe "after_create" do
+    before do
+      @post = associated_post
+      @user = authenticated_user(email_favorites: true)
+      @other_user = authenticated_user
+      @comment = Comment.new(body: 'My comment is really great', post: @post, user: @other_user)
+    end
+
+    it "sends an email to users who have favorited the post" do
+      favorite = @user.favorites.create(post: @post)
+
+      allow( FavoriteMailer )
+        .to receive(:new_comment)
+        .with(@user, @post, @comment)
+        .and_return( double(deliver_now: true))
+
+        expect( FavoriteMailer )
```

Comment:
This was copied from the checkpoint. Should we ask that this be modified?

On Tue, Jun 30, 2015 at 10:42 PM Bobbilee <notifications@github.com> wrote:

> In spec/models/comment_spec.rb
> <https://github.com/voltaaage/Bloccit/pull/21#discussion_r33645973>:
>
> > +    before do
> > +      @post = associated_post
> > +      @user = authenticated_user(email_favorites: true)
> > +      @other_user = authenticated_user
> > +      @comment = Comment.new(body: 'My comment is really great', post: @post, user: @other_user)
> > +    end
> > +
> > +    it "sends an email to users who have favorited the post" do
> > +      favorite = @user.favorites.create(post: @post)
> > +
> > +      allow( FavoriteMailer )
> > +        .to receive(:new_comment)
> > +        .with(@user, @post, @comment)
> > +        .and_return( double(deliver_now: true))
> > +
> > +        expect( FavoriteMailer )
>
> one line
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/voltaaage/Bloccit/pull/21/files#r33645973>.
>